### PR TITLE
[CLEANUP] Drop unused local variables

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Comment by oliverklee** [comment]
Do you need anything from me so you can merge this PR?

**Comment by oliverklee** [comment]
Rebased.

**Comment by oliverklee** [comment]
ping

**Comment by oliverklee** [comment]
ping

File: widgets/progressbar/class.tx_mkforms_widgets_progressbar_Main.php [hunk 1]
```diff
@@ -139,13 +133,8 @@ function initSteps() {
 	function getValue() {
 		$fValue = floatval(parent::getValue());
 
-		$fMin = $this->getMinValue();
 		$fMax = $this->getMaxValue();
 
-		if($fValue < $fMin) {
-			$mValue = $fMin;
```
Comment (hannesbochmann) [inline widgets/progressbar/class.tx_mkforms_widgets_progressbar_Main.php]: please dont remove this. This as to be fixed. $mValue should be changed to $fValue

Comment (oliverklee) [inline widgets/progressbar/class.tx_mkforms_widgets_progressbar_Main.php]: done

**Pull request merged**